{"version":3,"file":"notFoundError.js","sourceRoot":"/","sources":["util/Errors/Endpoints/notFoundError.ts"],"names":[],"mappings":";;;AAEA,MAAa,aAAc,SAAQ,KAAK;IAKpC,YAAa,OAAe;QAExB,KAAK,CAAE,OAAO,CAAE,CAAA;QAJb,eAAU,GAAW,GAAG,CAAA;QAK3B,IAAI,CAAC,OAAO,GAAG,OAAO,CAAA;QAGtB,MAAM,CAAC,cAAc,CAAC,IAAI,EAAE,GAAG,CAAC,MAAM,CAAC,SAAS,CAAC,CAAC;QAClD,KAAK,CAAC,iBAAiB,CAAC,IAAI,CAAC,CAAC;IAClC,CAAC;CAGJ;AAhBD,sCAgBC","sourcesContent":["\r\n\r\nexport class NotFoundError extends Error \r\n{\r\n\r\n    public statusCode: number = 404 \r\n\r\n    constructor( message: string )\r\n    {\r\n        super( message ) \r\n        this.message = message \r\n\r\n        // Settings \r\n        Object.setPrototypeOf(this, new.target.prototype); \r\n        Error.captureStackTrace(this); \r\n    }\r\n\r\n\r\n}"]}